packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -29,6 +29,19 @@ function parseTrigger(userFacingTrigger: NotificationTriggerInput): NativeNotifi
     return { type: 'date', timestamp: userFacingTrigger.getTime() };
   } else if (typeof userFacingTrigger === 'number') {
     return { type: 'date', timestamp: userFacingTrigger };
+  } else if (
+    // eg. { seconds: ..., repeats: ..., hour: ... }
+    ('seconds' in userFacingTrigger &&
+      'repeats' in userFacingTrigger &&
+      Object.keys(userFacingTrigger).length > 2) ||
+    // eg. { seconds: ..., hour: ... }
+    ('seconds' in userFacingTrigger &&
+      !('repeats' in userFacingTrigger) &&
+      Object.keys(userFacingTrigger).length > 1)
```

Comment:
What would you say about moving this condition to a separate function? It looks ugly with such formatting 😞